_posts/2023-10-11-14.0.0-release.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Apache Arrow 14.0.0 Release"
+date: "2023-10-11 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 14.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 13.0.0 release, Metehan Yildirim and Oleks V. have been invited to be committers.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
```

Comment:
@lidavidm

Reply:
```suggestion
## Arrow Flight RPC notes

A new RPC method was added to allow polling for completion in long-running queries as an alternative to the blocking GetFlightInfo call ([GH-36155](https://github.com/apache/arrow/issues/36155)). Also, `app_metadata` was added to `FlightInfo` and `FlightEndpoint` ([GH-37635](https://github.com/apache/arrow/issues/37635)).

In C++ and Python, an experimental asynchronous GetFlightInfo call was added to the client-side API ([GH-36512](https://github.com/apache/arrow/issues/36512)). `ServerCallContext` now exposes conveniences to send headers/trailers without having to use middleware ([GH-36952](https://github.com/apache/arrow/issues/36952)). The implementation was fixed to not reject unknown field tags to enable interoperability with future versions of Flight that could add new fields ([GH-36975](https://github.com/apache/arrow/issues/36975)). The CMake configuration was fixed to correctly require linking to Arrow Flight RPC when using Arrow Flight SQL ([GH-37406](https://github.com/apache/arrow/issues/37406)). 

In Go, the underlying generated Protobuf code is now exposed for easier low-level integrations with Flight ([GH-36893](https://github.com/apache/arrow/issues/36893)). 

In Java, the stateful "login" authentication APIs using the Handshake RPC are deprecated; it will not be removed, but it should not be used unless you specifically want the old behavior ([GH-37722](https://github.com/apache/arrow/issues/37722)). Utilities were added to help implement basic Flight SQL services for unit testing ([GH-37795](https://github.com/apache/arrow/issues/37795)).
```